app/config.py

```diff
@@ -0,0 +1,25 @@
+from dataclasses import dataclass
+from astropy.time import Time
+from astropy.units import Quantity
+
+
+@dataclass
+class CollectorConfig:
+    semesters: set
+    program_types: set
+    obs_classes: set
```

Comment:
We should make these `FrozenSet[Semester]`, etc.

Reply:
This is removed in #120